lib/asciidoctor/opal_ext/file.rb

```diff
@@ -68,6 +68,15 @@ def each_line(separator = $/, &block)
     end
   end
 
+  def readlines()
```

Comment:
It's generally accepted in the Ruby community to drop the brackets for methods that have no arguments.